aten/src/ATen/native/Correlation.cpp

```diff
@@ -0,0 +1,70 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+namespace native {
+
+Tensor cov(
+    const Tensor& self,
+    int64_t correction,
+    const c10::optional<Tensor>& fweights,
+    const c10::optional<Tensor>& aweights) {
+  TORCH_CHECK(
+      self.ndimension() <= 2, "cov(): input has more than 2 dimensions");
```

Comment:
suggested tweak: "cov(): expected input to have two or fewer dimensions but got an input with [dimensions] dimensions!"